element-templates/slack-connector.json

```
@@ -92,6 +117,31 @@
       },
       "constraints": {
         "notEmpty": true
+      },
+      "condition": {
+        "property": "method",
+        "equals": "chat.postMessage"
+      }
+    },
+    {
+      "label": "Channel Name",
+      "group": "createChannel",
+      "type": "String",
+      "feel": "optional",
+      "binding": {
+        "type": "zeebe:input",
+        "name": "data.name"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "pattern": {
+          "value": "^[a-z0-9-_]{1}[a-z0-9-_]{0,20}$",
```

Looking at the [API docs](https://api.slack.com/methods/conversations.create), I think you can use up to 80 chars for the name. I can also verify this in our Slack workspace manually. Plus, I think we can simplify the regex.
```suggestion
          "value": "^[-_a-z0-9]{1,80}$",
```